src/cli.ts

```diff
@@ -1,6 +1,6 @@
 import { Command, OptionValues } from 'commander'
 import { UnsignedTxJson, SignedTxJson, Context, ContextFile, FlareTxParams } from './interfaces'
-import { contextEnv, contextFile, getContext } from './constants'
+import { contextEnv, contextFile, getContext,colorCodes,networkMapping } from './constants'

```

Comment:
No spaces -> use prettier extension to fix these formatting errors automatically.